lib/Core/Executor.cpp

```diff
@@ -198,17 +198,27 @@ cl::opt<ExternalCallPolicy> ExternalCalls(
     cl::init(ExternalCallPolicy::Concrete),
     cl::cat(ExtCallsCat));
 
-cl::opt<bool> SuppressExternalWarnings(
-    "suppress-external-warnings",
-    cl::init(false),
-    cl::desc("Supress warnings about calling external functions."),
-    cl::cat(ExtCallsCat));
 
-cl::opt<bool> AllExternalWarnings(
-    "all-external-warnings",
-    cl::init(false),
-    cl::desc("Issue a warning everytime an external call is made, "
-             "as opposed to once per function (default=false)"),
+/*** External call warnings options ***/
+
+enum class ExtCallWarnings {
+  None,            // Never warn on external calls
+  OncePerFunction, // Only warn once per function on external calls (default)
```

Comment:
```suggestion
  OncePerFunction, // Only warn once per function on external calls
```

Reply:
The default is set by the option not here.

Reply:
Done